cmd/soroban-cli/src/commands/contract/invoke.rs

```diff
@@ -261,6 +221,59 @@ impl Cmd {
         Ok((function.clone(), spec, invoke_args, signers))
     }
 
+    pub async fn parse_arg(
+        &self,
+        input: &ScSpecFunctionInputV0,
+        matches_: &clap::ArgMatches,
+        config: &config::Args,
+        spec: &Spec,
+    ) -> Result<(ScVal, Option<StellarSigner>), Error> {
+        let mut signer: Option<StellarSigner> = None;
+        let name = input.name.to_utf8_string()?;
+        let sc_val = if let Some(mut val) = matches_.get_raw(&name) {
+            let mut s = val.next().unwrap().to_string_lossy().to_string();
+            if matches!(input.type_, ScSpecTypeDef::Address) {
+                if let Ok(signer_) = config
+                    .sign_with
+                    .locator
```

Comment:
We assuming here that sign_with have a locator (therefore has an access to a local fs), which is fine by me, as I think we only allow signing soroban authes with local key only (at least for now), but it would be nice to have a comment here